src/styles/blocks/header.scss

```diff
@@ -26,17 +24,22 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
-
     &:not(:last-child) {
       margin-right: 20px;
     }
   }
 
   &__link {
+    $hover-duration: 300ms;
     color: inherit;
     text-decoration: none;
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $hover-duration;
```

Comment:
Specify which properties the transition should apply to for better performance. For example, transition-property: color;